api/search/views.py

```diff
@@ -662,11 +662,12 @@ def search_fields(self):
         return [
             'abstract',
             'collectedType',
-            'contributors',
+            'contributors.fullname',
```

Comment:
Product has asked that we add all the name parts:
- `given_name`
- `middle_names`
- `family_name`

Reply:
Talked to product. Will defer this improvement as we don't index other name parts for nodes and preprints.